montreal-python/videos/lording-over-your-minions-with-salt-mp36.json

```diff
@@ -0,0 +1,41 @@
+{
+  "copyright_text": "Creative Commons Attribution license (reuse allowed)",
+  "description": "Montreal, April 8, 2013 -- Python developer Kamal Marhubi presents Salt -- a remote execution and configuration management tool written in Python. This talk is an introduction to salt, focusing on the configuration management aspects. it covers using salt with Vagrant as an alternative to Chef or Puppet. Kamal also demos using it to build up and control a machine on EC2 (network allowing).\n\nUseful links:\n\n* salt: http://sflx.ca/salt\n* Kamal: https://github.com/kmrhb\n* MP36: http://sflx.ca/mp36\n* Montreal Python: http://montrealpython.org\n* Savoir-faire Linux: http://www.savoirfairelinux.com",
```

Comment:
my account was renamed to kamalmarhubi

Reply:
Done in 9f35fce

Reply:
If you edit it in the GitHub editor, I think, then it becomes a pull request to my pull request.

Lowering the work and entry barrier is a main goal in pyvideo :)